docs/site/tutorials/Swift_autodiff_sharp_edges.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>Boolean <code>&amp;&amp;</code> and <code>||</code> should work normally now since https://github.com/apple/swift/pull/33483/ was merged, see e.g. https://github.com/efremale/swift/blob/b4764308b81421dabaa58db5008d350b628070bb/test/AutoDiff/validation-test/control_flow.swift#L810</p><p> </p><p>For example:</p><p>@differentiable</p><pre class="ql-syntax" spellcheck="false">func f(x: Float) -&gt; Float {
&nbsp;&nbsp;&nbsp;&nbsp;if x &gt; 0 &amp;&amp; x &lt; 10 {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return x
&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;else if x &lt;= 0 {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return 0
&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;else {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return x * x
&nbsp;&nbsp;&nbsp;&nbsp;}
}
print(gradient(at: 5, in: f)) // works, prints 1.0
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
@saeta, this isn't available in the Release version yet, right?

Reply:
Yeah the v0.11 Release can't differentiate through `||` and `&&`. Should I wait for the next Release version to have this feature before I remove `||` and `&&` from the tutorial, or remove it now?

Reply:
Waiting for the release sounds good to me.